dev.goats.xivlauncher.yml

```
@@ -0,0 +1,70 @@
+app-id: dev.goats.xivlauncher
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet6
+build-options:
+  build-args:
+    - --share=network
+  append-path: "/usr/lib/sdk/dotnet6/bin"
+  append-ld-library-path: "/usr/lib/sdk/dotnet6/lib"
+  env:
+    PKG_CONFIG_PATH: "/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/sdk/dotnet6/lib/pkgconfig"
+    DOTNET_CLI_TELEMETRY_OPTOUT: "true"
+command: /app/opt/XIVLauncher/XIVLauncher.Core
+modules:
+  - name: libsecret
+    buildsystem: meson
+    config-opts:
+      - "-Dmanpage=false"
+      - "-Dvapi=false"
+      - "-Dgtk_doc=false"
+    cleanup:
+      - /bin
+      - /include
+      - /lib/pkgconfig
+      - /share/gir-1.0
+      - /share/man
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/libsecret/0.20/libsecret-0.20.5.tar.xz
+        sha256: 3fb3ce340fcd7db54d87c893e69bfc2b1f6e4d4b279065ffe66dac9f0fd12b4d
+  - name: aria2
+    config-opts:
+      - '--with-ca-bundle=/etc/ssl/certs/ca-certificates.crt'
+    sources:
+      - type: archive
+        url: https://github.com/aria2/aria2/releases/download/release-1.36.0/aria2-1.36.0.tar.xz
+        sha256: 58d1e7608c12404f0229a3d9a4953d0d00c18040504498b483305bcb3de907a5
+  - name: xivlauncher
+    buildsystem: simple
+    build-commands:
+      - install -d /app/opt/XIVLauncher/
+      - dotnet publish -r linux-x64 --sc -o /app/opt/XIVLauncher/ /run/build/xivlauncher/src/XIVLauncher.Core/XIVLauncher.Core.csproj --configuration Release
```

you need to tell flathub this is x86_64 only https://github.com/flathub/flathub/wiki/App-Maintenance#only-build-on-x86_64

(since you mentionned WINE, I assume there is aarch64)

```suggestion
      - dotnet publish -r linux-x64 --sc -o ${FLAPTAK_DEST}/opt/XIVLauncher/ src/XIVLauncher.Core/XIVLauncher.Core.csproj --configuration Release
```